### ISSUE TITLE
video wallpaper without thumbnail does not work

### ISSUE THREAD

**Issue by arch-user-france1** [iabem97/komorebi#326]
If I create a new wallpaper (video or website) and don't add any thumbnail because I don't have any it simply doesn't show up in komorebi. If I add some random picture as a thumbnail it works most of the time.

**Comment by pendragons-code** [comment]
Yeah, because the thumbnail is what shows up in the UI, you can however, manually change the config in the files tho

**Comment by arch-user-france1** [comment]
I think the option should be required (none should not be possible). It should either generate a thumbnail if none is selected or use a default thumbnail.
